src/sources/util/http/auth.rs

```diff
@@ -11,7 +11,7 @@ use warp::http::HeaderMap;
 ))]
 use super::error::ErrorMessage;
 
-/// HTTP Basic authentication configuration.
+/// HTTP basic authentication configuration.
```

Comment:
Should this be capitalized? https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#basic_authentication_scheme